qir/qat/ValidationPass/ValidationPassConfiguration.cpp

```diff
@@ -61,7 +61,46 @@ namespace quantum
             profile.allowed_pointer_types_   = {"i8*",    "i16*",    "i32*",    "i64*",
                                               "Qubit*", "Qubit**", "Result*", "Result**"};
         }
-        else if (name == "quantinuum" || name == "rigetti" || name == "qci")
+        else if (name == "provider_4bf9")
+        {
+            profile.allow_internal_calls_     = false;
+            profile.allowlist_external_calls_ = true;
+            profile.allowlist_opcodes_        = true;
+            profile.opcodes_        = Set{"ret", "br", "phi", "add", "sub", "mul", "ashr", "and", "or", "xor"};
+            profile.external_calls_ = Set{
+                "__quantum__qis__cnot__body",        "__quantum__qis__cz__body",
+                "__quantum__qis__swap__body",        "__quantum__qis__h__body",
+                "__quantum__qis__s__body",           "__quantum__qis__s__adj",
+                "__quantum__qis__t__body",           "__quantum__qis__t__adj",
+                "__quantum__qis__x__body",           "__quantum__qis__y__body",
+                "__quantum__qis__z__body",           "__quantum__qis__rx__body",
+                "__quantum__qis__ry__body",          "__quantum__qis__rz__body",
+                "__quantum__qis__reset__body",       "__quantum__qis__mz__body",
+                "__quantum__qis__read_result__body",
+
+            };
+            profile.allowlist_pointer_types_ = true;
+            profile.allowed_pointer_types_   = {"Qubit*", "Result*"};
+        }
+
+        else if (name == "provider_7ee0")
+        {
+            // Supported LLVM types: i64 (integer), double, i1 (bool), %Qubit*, %Result*.
```

Comment:
I like having this commented here... could you add it to the above provider section as well?